RepoDb.Core/RepoDb/Extensions/DataEntityExtension.cs

```diff
@@ -36,9 +37,10 @@ public static IEnumerable<ClassProperty> GetProperties<TEntity>()
         // GetMappedName
         internal static string GetMappedName(Type type)
         {
-            return type.GetCustomAttribute<MapAttribute>()?.Name ??
-                ClassMapper.Get(type) ??
-                type.Name;
+            return type.GetCustomAttribute<TableAttribute>()?.Name ??
```

Comment:
Same here, `MapAttribute` must be in higher priority/precedence than `TableAttribute`?